git-coco

```diff
@@ -11,6 +11,34 @@ fail() {
     exit 1
 }
 
+url_parser() {
+    local url
+    local profile repo
+    local codecommit_regex="^codecommit::[a-z][a-z]-[a-z]+-[1-3]:\/\/"
+
+    url="$1"
+    
+    # codecommit:// url
+    if [[ $url =~ $codecommit_regex ]]; then
+        profile=$(echo $url | awk -F '[/@]' '{print $3}')
+        repo=$(echo $url | awk -F '[/@]' '{print $4}')
+
+        # If no profile set repo to what was parsed to profile otherwise set parsed profile to global profile
+        if [ -z "$repo" ]; then
+            repo=$profile
+        else
+            $_profile=$profile
+        fi
+
+    # https:// url
+    else
+        repo=$(echo $url | awk -F '/' '{print $NF}')
```

Comment:
I would quote the URL here (as well as verifying that it belongs to AWS CodeCommit, as noted below)